.babelrc.js

```diff
@@ -0,0 +1,33 @@
+const presets = [
+  "@babel/preset-flow",
+  [
+    "@babel/preset-env",
+    {
+      "targets": {
+        "browsers": "last 2 versions"
+      },
+      "loose": true,
+      "modules": false
+    }
+  ]
+];
+
+const plugins = [
+  "@babel/plugin-syntax-jsx",
+  "@babel/plugin-transform-react-jsx",
+  "@babel/plugin-proposal-class-properties"
+];
+
+module.exports = {
+  presets,
+  plugins
+
+  env: {
+    server: {
+      plugins: ["@babel/transform-modules-commonjs"]
+    },
+    test: {
+      plugins: ["@babel/transform-modules-commonjs"]
+    }
+  }
+}
```

Comment:
Note that now you can support the Babel configuration in JavaScript for your users.

I recommend you to use `process.env` directly instead of the `env` key here.